x-pack/plugins/alerting/server/authorization/alerting_authorization.ts

```diff
@@ -138,6 +142,41 @@ export class AlertingAuthorization {
     return this.authorization?.mode?.useRbacForRequest(this.request) ?? false;
   }
 
+  public async getAuthorizedAlertsIndices(featureIds: string[]): Promise<string[] | undefined> {
```

Comment:
Can we get unit tests for any new public functions that are added to this client?